src/commons/asBaseComponent.tsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 import {Appearance} from 'react-native';
-//@ts-ignore
 import hoistStatics from 'hoist-non-react-statics';
```

Comment:
noticed that `//@ts-ignore` is used in a lot of places due to undeclared `hoist-non-react-statics`, so I removed those.